clients/wallet_daemon_client/src/types.rs

```diff
@@ -1491,3 +1491,35 @@ pub struct WebauthnFinishAuthRequest {
     #[cfg_attr(feature = "ts", ts(type = "object"))]
     pub credential: PublicKeyCredential,
 }
+
+#[derive(Debug, Clone, Deserialize, Serialize)]
+#[cfg_attr(
+    feature = "ts",
+    derive(TS),
+    ts(export, export_to = "../../bindings/src/types/wallet-daemon-client/")
+)]
+pub struct TransferNftRequest {
+    pub nft_ids: Vec<NonFungibleId>,
```

Comment:
The NFT ID is not sufficient to point to a specific NFT - rather use NonFungibleAddress (resource + nft ID)

Reply:
Thanks for noticing, just pushed the new logic to use NFT addresses